core/record.go

```diff
@@ -308,6 +411,42 @@ func (t *Timetrace) loadFromRecordDir(recordDir string, filter ...func(*Record)
 
 outer:
 	for _, info := range filesInfo {
+		if filepath.Ext(info.Name()) == ".bak" {
```

Comment:
Here (and in the new `loadBackupsFromRecordDir` function I created in line 437) I implemented my own quickfix for point 1 of #161. I see that #162 has been merged already but I'm not sure how to pull those changes into my own branch - I tried `git fetch upstream`, `git pull`, but it did not seem to change my file. 

Is there a git way to manage this, short of copy-pasting manually the new code from #162? :sweat_smile:

Reply:
@jwnpoh If I understand correctly, all you have to do is to merge this repository's`main` branch into your branch.

---

To do so, this repository has to be an upstream repository of your local repository. In case you haven't done that yet, you can add this repository as an upstream like so:

```
git remote add original https://github.com/dominikbraun/timetrace
```

This will add my repository as an upstream called `original`. Then fetch my branches:

```
git fetch original
```

Now you can merge my `main` branch into your `delete-command` branch:

```
git merge original/main delete-command
```

This is also the typical workflow to keep your fork up-to-date, with the only difference that you would my `main` branch into your `main` branch.

Reply:
Thanks for the pointers. I believe the confilcts have been resolved and this PR is now as it is intended to be. Please have a look at the changes and let me know what you think!

Thanks!